data_steward/cdr_cleaner/cleaning_rules/sandbox_and_remove_pids.py

```diff
@@ -11,7 +11,7 @@
 SELECT t.* FROM `{{project}}.{{dataset}}.{{table}}` t
 {% if ehr_only and table not in ['death', 'aou_death'] %}
 JOIN `{{project}}.{{dataset}}._mapping_{{table}}` m
-ON t.{{table}}_id = m.{{table}}_id AND LOWER(m.src_hpo_id) != 'rdr'
+ON t.{{table}}_id = m.{{table}}_id AND LOWER(m.src_hpo_id) NOT IN ('ce', 'vibrent', 'healthpro')
```

Comment:
Are these constants somewhere? It would be helpful to reuse, but not a requirement

Reply:
We are going to add a mask to site maskings which can be utilized in mapping_tables once we are done with v8 generation.